src/index.ts

```diff
@@ -474,13 +475,16 @@ class SpellChecker {
   }
 
   load_dictionary() {
+    this.status_msg = 'Loading dictionary ...';
+    this.status_widget.update();
     return Promise.all([
       fetch(this.language.aff).then(res => res.text()),
       fetch(this.language.dic).then(res => res.text())
     ]).then(values => {
       this.dictionary = new Typo(this.language.name, values[0], values[1]);
-      console.log('Dictionary Loaded ', this.language.name, this.language.code);
+      // console.log('Dictionary Loaded ', this.language.name, this.language.code);
```

Comment:
Maybe just change to `console.debug`? The overhead is negligible but it can come in handy in the future.

Reply:
I agree, this looks good and is very handy!

Reply:
Should we do a new release from this PR or should we collect some more changed until we create a release? Do we have a placeholder in the beginning for something unreleased?

Reply:
I think that waiting about a week between releases is usually a good thing so that if we have a few other small changes we do not spam users who subscribe to release notifications with dozens of messages (and do not discourage them from upgrading regularly). I tend to do releases on weekends so that it does not interfere with my day job. As for placeholder: I would call it `### 0.6.0 (unreleased)` (minor version bump because we got a new feature, but not major as it does not break the API or change the old user settings).

Reply:
OK, I agree ... I'll do the most work over the week. The weekend is for the family ;-) But the plan to have the release on weekends is good!